tools/FindPythonLibsNew.cmake

```diff
@@ -1,84 +1,74 @@
-# - Find python libraries
-# This module finds the libraries corresponding to the Python interpreter
-# FindPythonInterp provides.
-# This code sets the following variables:
+# * Find python libraries This module finds the libraries corresponding to the Python interpreter
+#   FindPythonInterp provides. This code sets the following variables:
 #
-#  PYTHONLIBS_FOUND           - have the Python libs been found
-#  PYTHON_PREFIX              - path to the Python installation
-#  PYTHON_LIBRARIES           - path to the python library
-#  PYTHON_INCLUDE_DIRS        - path to where Python.h is found
-#  PYTHON_MODULE_EXTENSION    - lib extension, e.g. '.so' or '.pyd'
-#  PYTHON_MODULE_PREFIX       - lib name prefix: usually an empty string
-#  PYTHON_SITE_PACKAGES       - path to installation site-packages
-#  PYTHON_IS_DEBUG            - whether the Python interpreter is a debug build
+# PYTHONLIBS_FOUND           - have the Python libs been found PYTHON_PREFIX - path to the Python
+# installation PYTHON_LIBRARIES           - path to the python library PYTHON_INCLUDE_DIRS        -
+# path to where Python.h is found PYTHON_MODULE_EXTENSION    - lib extension, e.g. '.so' or '.pyd'
+# PYTHON_MODULE_PREFIX       - lib name prefix: usually an empty string PYTHON_SITE_PACKAGES - path
+# to installation site-packages PYTHON_IS_DEBUG            - whether the Python interpreter is a
+# debug build
```

Comment:
Yep! It tries to reformat moments by default. Nice for wrapping, bad for structures it doesn't recognize. I'll tone it back a bit here. I tried to check for this, but missed this one.

Reply:
I've turned off comment reformatting for now. We might be able to reactivate it at some point, I assume it can parse help headers like this one, just have to see what's off. But that's for later.